hermes/src/store/proof/batch_vaa.rs

```diff
@@ -97,6 +96,18 @@ pub fn get_price_feeds_with_update_data(
 }
 
 
+pub fn get_price_feed_ids(state: State) -> Vec<PriceIdentifier> {
```

Comment:
This could be cleaner with `try_map` which can collapse errors thrown during iteration.